apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -296,29 +303,41 @@ export const FileBrowser = React.memo(() => {
 
   return (
     <div className="file-browser">
-      <div className="files-toolbar">
-        <button type="button" onClick={() => newFolderPrompt()}>
-          <i className="fa-solid fa-folder" />
-          &nbsp; New Folder
-        </button>
-        <button type="button" onClick={() => newFilePrompt()}>
-          <i className="fa-solid fa-file" />
-          &nbsp; New File
-        </button>
-      </div>
-      <ul>
-        <InnerFileBrowser
-          parentId={DEFAULT_FOLDER_ID}
-          folders={project.folders}
-          newFolderPrompt={newFolderPrompt}
-          files={project.files}
-          newFilePrompt={newFilePrompt}
-          moveFilePrompt={moveFilePrompt}
-          renameFilePrompt={renameFilePrompt}
-          renameFolderPrompt={renameFolderPrompt}
-          toggleFileVisibility={toggleFileVisibility}
-        />
-      </ul>
+      <PanelContainer
+        id="file-browser"
+        headerContent={'Files'}
+        className={moduleStyles['file-browser']}
+        rightHeaderContent={
+          <PopUpButton iconName="plus" alignment="right">
```

Comment:
maybe worth extracting this into a helper method for readability?

Reply:
I broke it out into a separate `FileBrowserHeaderPopUpButton` component.